calculator.cpp

```diff
@@ -2,26 +2,32 @@
 using namespace std;
 
 int addition(int number_1, int number_2){
-
+return number_1 + number_2;
 }
 
 int subtraction(int number_1, int number_2){
-
+return number_1 - number_2;
 }
 
 int multiplication(int number_1, int number_2){
-
+return number_1*number_2;
 }
 
 int division(int number_1, int number_2){
-
+return number_1/number_2;
 }
 
 int get_input(){
-
+int a,b;
+cout<<"\n Enter two Integers :: ";
+cin>>a>>b;
+cout<<"\n Sum of Numbers : "<<addition(a,b);
```

Comment:
Displaying the numbers as well will improve user experience.
Sum of Numbers 6 and 5 is 11. Something like this.

Reply:
Thanks for the review. Will do the updates.